contrib/datawave-quickstart/bin/services/accumulo/bootstrap.sh

```diff
@@ -117,7 +117,61 @@ function accumuloStop() {
 }
 
 function accumuloStatus() {
-    accumuloIsRunning && echo "Accumulo is running. PIDs: ${DW_ACCUMULO_PID_LIST} ${DW_ZOOKEEPER_PID_LIST}" || echo "Accumulo is not running"
+    # define vars for accumulo processes
+    local _gc
+    local _master
+    local _monitor
+    local _tracer
+    local _tserver
+
+    echo "======  Accumulo Status  ======"
+    local _opt=pid
+    local _arg
+
+    accumuloIsRunning
+    test -n "${DW_ACCUMULO_PID_LIST}" && {
+        local -r _pids=${DW_ACCUMULO_PID_LIST// /|}
+        echo "pids: ${DW_ACCUMULO_PID_LIST}"
+
+        for _arg in $(jps -lm | egrep "${_pids}"); do
+            case ${_opt} in
+                pid)
+                    _pid=${_arg}
+                    _opt=class;;
+                class) _opt=component;;
+                component)
+                    local _none
+                    case "${_arg}" in
+                        gc) _gc=${_pid};;
+                        master) _master=${_pid};;
+                        monitor) _monitor=${_pid};;
+                        tracer) _tracer=${_pid};;
+                        tserver) _tserver=${_pid};;
+                        *) _none=true;;
+                    esac
+
+                    test -z "${_none}" && info "${_arg} => ${_pid}"
+                    _opt=address
+                    unset _none
+                    _pid=;;
+                address) _opt=addrValue;;
+                addrValue) _opt=pid;;
+            esac
+        done
+    }
+
+    test -z "${_gc}" && error "accumulo gc is not running"
+    test -z "${_master}" && error "accumulo master is not running"
+    test -z "${_monitor}" && error "accumulo monitor is not running"
+    test -z "${_tracer}" && error "accumulo tracer is not running"
```

Comment:
Having a tracer running is not a requirement for a usable accumulo system.  Neither is the monitor for that matter.  I believe all you need is a master, gc, and 1 or more tservers for the system to be up and usable.
Event then, if the gc is not running or the master is not running one can use the system.  Just saying....

Reply:
At a minimum specifying which pieces of the system are not running like you do for the datawave status below would be useful.

Reply:
will make these info instead of error